src/main/java/com/signicat/hystrix/servlet/AsyncWrapperServlet.java

```diff
@@ -117,7 +117,7 @@ public void service(final HttpServletRequest req, final HttpServletResponse resp
 
         //double thread pool size for pool with magic name 'default'
         final int maximumSize = DEFAULT_COMMAND_GROUP_KEY.equals(key) ? (corePoolSize * 2) : corePoolSize;
-        final int coreSize    = Math.min(10, maximumSize);
+        final int coreSize    = Math.max(10, maximumSize);
```

Comment:
max? really?

Reply:
yeah, pretty sure, i agree it's strange. i noticed portal behaving differently, and had another look at the original code.

Reply:
This is easily unit testable for various values

Reply:
yeah, please ignore that, i was just seeing double for a moment